radio/bluetooth/bluetooth.go

```diff
@@ -102,11 +161,11 @@ func Scan(id string, timeout time.Duration) (map[string]bool, error) {
 	return devices, nil
 }
 
-func Online(id string, timeout time.Duration) (bool, error) {
+func Online(id string) (bool, error) {
 	online := false
 	advChannel := make(chan ble.Advertisement)
 	ctx, cancel := context.WithCancel(context.Background())
-	ctx = ble.WithSigHandler(context.WithTimeout(ctx, timeout*time.Second))
+	ctx = ble.WithSigHandler(context.WithTimeout(ctx, longTimeout*time.Second))
```

Comment:
It looks like you always use `longTimeout` and `shortTimeout` in conjunction with `* time.Second` so it would probably make more sense just to store `longTimeout` and `shortTimeout` as the multiplied value.